### ISSUE TITLE
Helm install project fails with secrets argument error on project install

### ISSUE THREAD

**Issue by squillace** [Azure/brigade#131]
Using the default brigade installation from helm (not the helm repo version), I attempt to install a project per https://github.com/Azure/brigade#quickstart and I get this:

```
➜  brigade git:(master) helm install chart/brigade-project -f ~/work/nogo/slacid/empty-testbed.yaml --name slack-thingie
Error: render error in "brigade-project/templates/secret.yaml": template: brigade-project/templates/secret.yaml:18:38: executing "brigade-project/templates/secret.yaml" at <b64enc>: wrong number of args for b64enc: want 1 got 0
```

this for a project that has consistently worked. related to #130

**Comment by technosophos** [comment]
I just reproduced this one. I'll fix it.

**Comment by technosophos** [comment]
Closed via #136
